README.md

```diff
@@ -1,59 +1,58 @@
-================================================================================                            
-                              Cargo Tracker
-            Applied Domain-Driven Design Blue Prints for Java EE
-================================================================================
+Cargo Tracker: Applied Domain-Driven Design Blue Prints for Java EE
+=======
+
+# Overview
 
-Overview
-========
 The project demonstrates how you can develop applications with the Java EE 
 platform using widely adopted architectural best practices like Domain-Driven 
 Design (DDD), TDD and agile. The project is directly based on the well known 
 original Java DDD sample (http://dddsample.sourceforge.net) application 
 developed by DDD pioneer Eric Evans' company Domain Language and the Swedish 
 software consulting company Citerus. The cargo example actually comes from 
 Eric Evans' seminal book on DDD. The original application is written in Spring,
-Hibernate and Jetty whereas the application is build entirely on Java EE 8 and
-GlassFish 4.  It also supports WildFly 8.
+Hibernate and Jetty whereas the application is build entirely on Java EE 7 and
+Payara 4.
 
 The application is an end-to-end system for keeping track of shipping cargo. It 
 has several interfaces described in the following sections.
 
 For further details on the project, please visit:
 https://github.com/eclipse-ee4j/cargotracker
-Getting Started
-===============
+ 
+# Getting Started
+
 The project site has detailed information on how to get started: 
 https://projects.eclipse.org/projects/ee4j.cargotracker/.
 
 The simplest steps are the following (no IDE required):
 
-* Get the project source code: https://github.com/eclipse-ee4j/cargotracker
+* Get the project source code: `git clone https://github.com/eclipse-ee4j/cargotracker.git`
 * As long as you have Maven set up, navigate to the project source root and 
-  type: mvn package cargo:run
-* To run with WildFly 8x, type: mvn -Pwildfly package cargo:run
+  type: `mvn package cargo:run`
+* To run with WildFly 8x, type: `mvn -Pwildfly package cargo:run`

```

Comment:
Let's please remove WildFly support. That work was never done properly. We can revisit it later.